src/DynamoCoreWpf/Views/PackageManager/PublishPackageView.xaml

```diff
@@ -132,7 +132,6 @@
                                                 <ContentPresenter Name="ContentSite"
                                                                   IsHitTestVisible="False"
                                                                   Content="{Binding Path=SelectedHostsString, Mode=OneWay, UpdateSourceTrigger=PropertyChanged}"
-                                                                  ContentTemplateSelector="{TemplateBinding ItemTemplateSelector}"
```

Comment:
was this not used?

Reply:
@mjkkirschner Yes since I am not using a template for content but instead writing my own binding